lib/plugins/aws/deploy/lib/existsDeploymentBucket.test.js

```diff
@@ -68,24 +68,25 @@ describe('#existsDeploymentBucket()', () => {
     });
   });
 
-  [{ region: 'eu-west-1', response: 'EU' }, { region: 'us-east-1', response: '' }].forEach(
-    value => {
-      it(`should handle inconsistent getBucketLocation responses for ${value.region} region`, () => {
-        const bucketName = 'com.serverless.deploys';
-
-        awsPlugin.provider.options.region = value.region;
-
-        sinon.stub(awsPlugin.provider, 'request').resolves({
-          LocationConstraint: value.response,
-        });
-
-        awsPlugin.serverless.service.provider.deploymentBucket = bucketName;
-        return expect(awsPlugin.existsDeploymentBucket(bucketName)).to.be.fulfilled.then(() => {
-          expect(awsPluginStub.args[0][0]).to.equal('S3');
-          expect(awsPluginStub.args[0][1]).to.equal('getBucketLocation');
-          expect(awsPluginStub.args[0][2].Bucket).to.equal(bucketName);
-        });
+  [
```

Comment:
I believe this change is not intended (and few others, I will comment on).

I take you've run `npm run prettify` (or prettier in general no whole codebase ?)

To narrow prettification just to committed changes run `npm run prettify-updated`

Reply:
I run `npm run prettify` command, thanks for checking these unintended changes. (I have reverted them).